examples/quickstart_gcp/quickstart_gcp/io_managers.py

```diff
@@ -46,10 +46,11 @@ def handle_output(self, context: OutputContext, obj: pd.DataFrame):
 
     def load_input(self, context: InputContext):
         # upstream_output.asset_key is the asset key given to the Out that we're loading for
-        table_name = context.upstream_output.asset_key.to_python_identifier()
+        table_name = context.asset_key.to_python_identifier()
```

Comment:
is `context.upstream_output.asset_key` and `context.asset_key` guaranteed to be the same in all cases?

Reply:
I think so but inquiring with @sryza for confirmation on this, who suggested `context.asset_key`

Reply:
Yes - it should be.